backend/src/providers/oauth/apple.provider.ts

```
@@ -186,7 +186,10 @@ export class AppleOAuthProvider implements OAuthProvider {
   /**
    * Verify Apple ID token and extract user info
    */
-  async verifyIdToken(idToken: string): Promise<AppleUserInfo> {
+  async verifyIdToken(
+    idToken: string,
+    audience?: string
+  ): Promise<AppleUserInfo> {
```

ESLint

[error] Replace `⏎····idToken:·string,⏎····audience?:·string⏎··` with `idToken:·string,·audience?:·string`

(prettier/prettier)